pkg/datastore/test/caveat.go

```diff
@@ -146,6 +146,17 @@ func WriteCaveatedRelationshipTest(t *testing.T, tester DatastoreTester) {
 	req.NoError(err)
 	assertTupleCorrectlyStored(req, ds, rev, tpl)
 
+	// RelationTupleUpdate_DELETE ignores caveat part of the request
```

Comment:
I still think we should make it *error* if the caveat is provided on the delete. Thoughts?

Reply:
well, what happens if the original relationship has a caveat, and the user is just sending it _as is_. You'd be forcing them to do extra work, and clearly from the client perspective they are sending "the exact same tuple we just stored". I'm not sold on this idea.

Reply:
True... but I could also see people being confused by sending the non-caveated rel for a caveated one and then being confused also

Reply:
what would confuse them? the rel would be deleted, which is what they would want in the first place. I can understand it feels off to send data that is not used at all as part of the method, but it seems like the least bad of the two options to me.